modules/sdk-coin-osmo/src/index.ts

```diff
@@ -2,3 +2,4 @@ export * from './lib';
 export * from './osmo';
 export * from './tosmo';
 export * from './register';
+export * as AbstractCosmos from '@bitgo/abstract-cosmos';
```

Comment:
The idea is to not have Abstract Cosmos visible to WP. Instead, whatever is required should be directly imported from coin sdk. We can handle it later as discussed.
Please mark a TODO comment.

Reply:
Added